pkg/start/start.go

```diff
@@ -103,6 +121,15 @@ func (o *Options) Run() error {
 	if o.ReleaseImage == "" {
 		return fmt.Errorf("missing --release-image flag, it is required")
 	}
+	if o.ServingCertFile == "" && o.ServingKeyFile != "" {
+		klog.Errorf("--serving-key-file was set, so --serving-cert-file must also be set")
+	}
+	if o.ServingKeyFile == "" && o.ServingCertFile != "" {
+		klog.Errorf("--serving-cert-file was set, so --serving-key-file must also be set")
+	}
+	if o.ServingKeyFile == "" {
+		klog.Errorf("missing --serving-key-file flag, it is required")
```

Comment:
It is not required.  Leaving both `--serving-...` options unset just means "don't server HTTPS".  That's valid, and is what we want to do during CVO bootstrapping.  I think we should drop this `if` block.